jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/SingletonRuntimeManager.java

```diff
@@ -237,14 +237,19 @@ public void close() {
         this.singleton = null;   
     }
     
+    private File getPersistedFile(String location, String identifier) {
+        // if windows, replace ':' by '_'
+        return new File(location + File.separatorChar +
+                        (File.separatorChar == '/' ? identifier : identifier.replace(':', '_')) + "-jbpmSessionId.ser");
```

Comment:
we have  commons lang3 in kie-server distribution. I would feel more comfortable using something like

https://commons.apache.org/proper/commons-lang/javadocs/api-3.4/src-html/org/apache/commons/lang3/SystemUtils.html#line.1273

wdyt ?

Reply:
yes, that seems safer. Thanks for pointing out that commons-lang was available.